crates/typos-cli/src/bin/typos-cli/main.rs

```diff
@@ -316,6 +317,8 @@ fn run_checks(args: &args::Args) -> proc_exit::ExitResult {
         if status_reporter.errors_found() {
             errors_found = true;
         }
+        
+        status_reporter.finalize().unwrap();
```

Comment:
We need to find something better to do than `unwrap`

Reply:
I just have no idea, ? can not cast error automatic, should i attach a map_err for everyone?

Reply:
Why is this marked as resolved when nothing has changed?

Reply:
I forgeted to update this one, will fixed in the new patch.

Reply:
> log::error!("Error finalizing: {}", err);

Is not an appropriate way of handling this